compilerlibs/Makefile.compilerlibs

```diff
@@ -25,14 +25,14 @@
 # that they are consistent with the interface digests in the archives.
 
 UTILS=utils/config.cmo utils/build_path_prefix_map.cmo utils/misc.cmo \
-	utils/identifiable.cmo utils/numbers.cmo utils/arg_helper.cmo \
-	utils/clflags.cmo utils/profile.cmo utils/local_store.cmo \
-	utils/load_path.cmo \
-	utils/terminfo.cmo utils/ccomp.cmo utils/warnings.cmo \
-	utils/consistbl.cmo utils/strongly_connected_components.cmo \
-	utils/targetint.cmo utils/int_replace_polymorphic_compare.cmo \
-	utils/domainstate.cmo utils/binutils.cmo utils/lazy_backtrack.cmo \
-        utils/diffing.cmo
+  utils/identifiable.cmo utils/numbers.cmo utils/arg_helper.cmo \
+  utils/clflags.cmo utils/profile.cmo utils/local_store.cmo \
+  utils/load_path.cmo \
+  utils/terminfo.cmo utils/ccomp.cmo utils/warnings.cmo \
+  utils/consistbl.cmo utils/strongly_connected_components.cmo \
+  utils/targetint.cmo utils/int_replace_polymorphic_compare.cmo \
+  utils/domainstate.cmo utils/binutils.cmo utils/lazy_backtrack.cmo \
+  utils/diffing.cmo
```

Comment:
Actually my favorite indentation style for such listings is to have one entry per line, to reduce the diff noise when we add new entries. If we're touching this anyway, would you maybe consider moving to that? (If you want to feel fancy with an `addsufix` that's also fine with me, but not a requirement.)

Reply:
I'll be fancy with **`addprefix`** in a sec 😉

Reply:
Actually, I've gone with the single dep per line, but not with a macro - they're not always the same and I think it's clearer therefore not to use `$(addprefix ` for any of them. It looks better with one-per-line too!